go/libkb/util.go

```diff
@@ -1065,7 +1065,9 @@ func FindPreferredKBFSMountDirs() (mountDirs []string) {
 }
 
 var kbfsPathInnerRegExp = func() *regexp.Regexp {
-	const socialAssertion = `[-_a-zA-Z0-9.]+@[a-zA-Z.]+`
+	const regularAssertion = `[-_a-zA-Z0-9.+]+@[a-zA-Z.]+`
+	const atContainingAssertion = `\[[-_a-zA-Z0-9.]+@[-_a-zA-Z0-9.]+\]@[a-zA-Z.]+`
```

Comment:
Does this only cover the email case? If so would it be better to just put that in the regex?

Also I know I didn’t put any comment before, but would you mind adding some examples in the comment for the new case?

Reply:
I was trying to be mildly future proof / the server accepts assertions that match this regex so I figured we should here too. Will add comments.